core/utils/dmljob.py

```diff
@@ -40,6 +40,17 @@ def __init__(
         self.weights = weights
         self.hyperparams = hyperparams
         self.label_column_name = label_column_name
+        self.list_of_serialized_weights = []
```

Comment:
This shouldn't be here. This seems to be for the averaging PR.

Reply:
done